src/electionguard/directory.c

```diff
@@ -0,0 +1,57 @@
+
+
+#include <linux/limits.h>     /* PATH_MAX */
```

Comment:
Build fails on Windows and Mac trying to include this header. 
Can we use `FILENAME_MAX` here from `stdio.h` instead? (that's also what we're using in other places where we're allocating a string for a path)